masterlist.yaml

```
@@ -20395,6 +20395,26 @@ plugins:
       - crc: 0x3DC7A6FB
         util: 'TES4Edit v4.0.4c'
 
+  - name: 'FrostcragAlchemySorter.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/13782/'
+        name: 'Frostcrag Alchemy Sorter'
+      - link: 'https://www.nexusmods.com/oblivion/mods/49969/'
+        name: 'Frostcrag Alchemy Sorter Updated'
+    msg:
+      - <<: *patchUpdateAvailable
+        type: warn
+        subs: [ '[Frostcrag Alchemy Sorter Updated](https://www.nexusmods.com/oblivion/mods/49969/)' ]
+        condition: 'version("FrostcragAlchemySorter.esp", "1.0", <)'
+    dirty:
+      - <<: *quickClean
+        crc: 0x192067D2
+        util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 2
+    clean:
+      - crc: 0xE855599E
```

```suggestion
      - crc: 0xBD3801E3
```

As the new updated version has a different checksum, the cleaning data should be updated.

---

Could we keep both checksums? Especially given that the 1.0 re-release was just put out today.

---

It's just a clean checksum, so it's not really worth it. Besides, the message will encourage users of the previous version to update.

---

Okay.